nautobot/docs/release-notes/version-2.4.md

```diff
@@ -98,6 +98,10 @@ Nautobot 2.4 adds the data models [`WirelessNetwork`](../user-guide/core-data-mo
 
 Refer to the [Wireless](../user-guide/core-data-model/wireless/index.md) documentation for more details.
 
+#### Apps Marketplace Page and Installed Apps Page Tile View
+
+Nautobot 2.4 introduces Apps Marketplace page containing information about uninstalled Nautobot apps. In addition to that, Installed Apps page is getting a brand-new tile view as an alternative to already existing list view.
```

Comment:
Perhaps "...information about available but not installed Nautobot Apps"?

Reply:
Done as part of a95d5e0dfa4d691af4a6b0a3172a19c6ea4e92da